apps/public/src/mocks/endpoints.ts

```diff
@@ -0,0 +1,3 @@
+export const PDOK_REVERSE = 'https://api.pdok.nl/bzk/locatieserver/search/v3_1/reverse'
```

Comment:
Seeing as we're using it as an object, why not make it an object here?

Reply:
sure!